modules/ossm-control-plane-deploy.adoc

```diff
@@ -84,21 +79,45 @@ Follow this procedure to deploy the {ProductName} control plane the command line
 ----
 $ oc login https://{HOSTNAME}:6443
 ----
-
++
 . Create a project named `istio-system`.
 +
 [source,terminal]
 ----
 $ oc new-project istio-system
 ----
-
-. Create a `ServiceMeshControlPlane` file named `istio-installation.yaml` using the example found in "Customize the {ProductName} installation". You can customize the values as needed to match your use case.  For production deployments you _must_ change the default Jaeger template.
-
-. Run the following command to deploy the control plane:
++
+. Create a `ServiceMeshControlPlane` file named `istio-installation.yaml` using the following example in "Customize the {ProductName} installation". For production deployments you _must_ customize the default Jaeger template. For more information, see **Customizing Jaeger deployment** in the Jaeger documentation.
```

Comment:
"Create a `ServiceMeshControlPlane` file named `istio-installation.yaml` using the following example in "Customize the {ProductName} installation"."

Are we using the following example, or the one from the Customizing topic?  Also, it's confusing that this is "Example 2", is that for version 2.0  or is it the second example?

Reply:
Re: Jaeger template, it looks like this example 2 is for non-production deployments (the default), since the Jaeger storage is set to Memory.  Having the note in line 99 about changing the template, and then not having the template in the YAML might be confusing.   This YAML file includes the Jaeger template configuration option.
https://github.com/maistra/istio-operator/blob/maistra-2.0/deploy/examples/maistra_v1_servicemeshcontrolplane_cr_auth.yaml